src/test/java/com/influxdb/v3/client/InfluxDBClientTest.java

```diff
@@ -167,10 +167,66 @@ public void testQuery() throws Exception {
                           Assertions.assertThat(objects[3].getClass()).isEqualTo(String.class);
                           Assertions.assertThat(objects[3]).isEqualTo("intel");
 
+                          Assertions.assertThat(objects[4].getClass()).isEqualTo(String.class);
+                          Assertions.assertThat(objects[4]).isEqualTo("empty");
+
                           Assertions.assertThat(objects[7].getClass()).isEqualTo(BigInteger.class);
                           Assertions.assertThat(objects[7]).isEqualTo(BigInteger.valueOf(timestamp * 1_000_000_000));
                       });
             }
         }
     }
+
+    @EnabledIfEnvironmentVariable(named = "TESTING_INFLUXDB_URL", matches = ".*")
+    @EnabledIfEnvironmentVariable(named = "TESTING_INFLUXDB_TOKEN", matches = ".*")
+    @EnabledIfEnvironmentVariable(named = "TESTING_INFLUXDB_DATABASE", matches = ".*")
+    @Test
+    public void testQueryRows() throws Exception {
```

Comment:
The test is named `testQueryRows()` plural, however it appears only 1 row is checked.  It might be slightly more reliable to check a stream with multiple (three or a dozen) records.

It also might be useful to test a basic exceptional use case.  For example when a query returns no or empty results or when an exception is returned from the server, which can happen when a passed query is malformed or otherwise unprocessable.